go/vt/vtctl/localvtctldclient/client.go

```diff
@@ -26,9 +26,8 @@ import (
 )
 
 var (
-	m               sync.RWMutex
-	server          vtctlservicepb.VtctldServer
-	errStreamClosed = errors.New("stream is closed for sending") // nolint (TODO:@ajm188) this will be used in a future PR, and the codegen will produce invalid code for streaming rpcs without this
+	m      sync.RWMutex
```

Comment:
Sorry I didn't notice this other `RWMutex` in the first review.
Should this also change to a plain Mutex?

Reply:
yeah, we can change that as well